os/arch/arm/include/bcm4390x/memory_region.h

```diff
@@ -0,0 +1,25 @@
+/****************************************************************************
+ *
+ * Copyright 2023 Samsung Electronics All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied. See the License for the specific
+ * language governing permissions and limitations under the License.
+ *
+ ****************************************************************************/
+
+extern uint32_t _sbss;
+extern uint32_t _ebss;
+extern uint32_t _sdata;
+extern uint32_t _edata;
+int mem_variable_region_count = 2;
+void *variable_region_start_addr[2] = { &_sbss, &_sdata };
+void *variable_region_end_addr[2] = { &_ebss, &_edata };
```

Comment:
How about using a definition instead the variable?
```
#define MEM_VAR_REG_COUNT 2
void *va......addr[MEM_VAR_REG_COUNT] = .....
...
```
Then, we can save a memory (no need a varaible) and no need extern.